test_elasticsearch_async/test_connection.py

```diff
@@ -58,7 +58,7 @@ def test_error_is_properly_logged(connection, caplog, port, server):
 def test_timeout_is_properly_raised(connection, server):
     @asyncio.coroutine
     def slow_request():
-        yield from asyncio.sleep(0.01)
+        yield from asyncio.sleep(1)
```

Comment:
it is due to the `TimeoutHandle` in aiohttp that uses ceil : 

    def start(self):
        if self._timeout is not None and self._timeout > 0:
            at = ceil(self._loop.time() + self._timeout)
            return self._loop.call_at(at, self.__call__)

The `ClientSession` requests timeout is called `read_timeout`.

Do you want another PR ?